xml/Accessibility/CAccPropServices.xml

```diff
@@ -2,6 +2,8 @@
   <TypeSignature Language="C#" Value="public interface CAccPropServices : Accessibility.IAccPropServices" />
   <TypeSignature Language="ILAsm" Value=".class public interface auto ansi abstract CAccPropServices implements class Accessibility.IAccPropServices" />
   <TypeSignature Language="DocId" Value="T:Accessibility.CAccPropServices" />
+  <TypeSignature Language="VB.NET" Value="Public Interface CAccPropServices&#xA;Implements IAccPropServices" />
+  <TypeSignature Language="F#" Value="type CAccPropServices = interface&#xA;    interface IAccPropServices" />
```

Comment:
```
type CAccPropServices = 
    interface
        interface IAccPropServices
    end
```

Reply:
So this is a bit tricky.  The "right" way to do this in modern F# is as follows:

```fsharp
type CAccPropServices =
    interface IAccPropServices with
        member_.ClearHwndProps(hwnd, idObject, idChild, .....)
```

However, given that interface members are not listed in MSDN either, the best way to show this is probably like this:

```fsharp
type CAccPropServices =
    interface IAccPropServices with
        ...
```
Currently, it renders like this:

```fsharp
type CAccPropServices = interface
    interface IAccPropServices
```

Which is not correct.